livekit/src/room/participant/local_participant.rs

```diff
@@ -643,17 +637,10 @@ impl LocalParticipant {
         self.inner.info.read().kind
     }
 
-    pub async fn perform_rpc(
-        &self,
-        destination_identity: String,
-        method: String,
-        payload: String,
-        response_timeout_ms: Option<u32>,
-    ) -> Result<String, RpcError> {
-        let response_timeout = Duration::from_millis(response_timeout_ms.unwrap_or(10000) as u64);
+    pub async fn perform_rpc(&self, params: PerformRpcParams) -> Result<String, RpcError> {
         let max_round_trip_latency = Duration::from_millis(2000);
```

Comment:
PerformRpcParams -> RpcData?

Reply:
I don't like it because I think we need to be clear this is used on the perform side, in contrast to RpcInvocationData which is used on the receive side...

Reply:
(still could be PerformRpcData if that feels more correct)